package.json

```diff
@@ -23,11 +23,11 @@
     "compile:electron": "webpack --progress --config webpack.electron.config.js",
     "compile:web": "webpack --config webpack.web.config.js",
     "compile": "yarn compile:electron && yarn compile:web",
-    "dev:electron": "webpack-dev-server --hot --progress --config webpack.electron.config.js",
-    "dev:web": "webpack-dev-server --hot --progress --config webpack.web.config.js",
+    "dev:electron": "cross-env webpack-dev-server --hot --progress --config webpack.electron.config.js",
```

Comment:
What is `cross-env` providing without flags?
I'd suggest setting `NODE_ENV=development` or removing it.

Reply:
Whoops. I ended up removing that, but never removed `cross-env`